### DISCUSSION TITLE
Windows 7

### DISCUSSION THREAD

**Discussion by harryray33** [dbeaver/dbeaver#18956] [Q&A] [answered]
Does this, or an earlier version, work with Win7?

Thanks

**Comment by E1izabeth** [comment, marked as answer]
Yes, it works.

**Comment by guy-rouillier** [comment]
Found via search.  I'm still using Windows 7, 64-bit, Professional.
dbeaver-ce-25.1.0-win32.win32.x86_64.zip works.
dbeaver-ce-25.1.2-win32.win32.x86_64.zip does not work.  Starts up the logo banner, but then the banner simply disappears.  I can't find any relevant messages in any of the Windows message logs.

**Reply by E1izabeth** [reply]
DBeaver stopped support for Windows 7 since 25.1.2

**Reply by guy-rouillier** [reply]
Thank you for clarifying.

**Reply by tony9527167** [reply]
25.1.0 is the latest version works on Windows 7, 
25.1.1 also does not works any more.
